client/charts/js/components/IncidentsBubbleMap.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {
+	filterDatasets,
+	groupByState,
+	groupByCity,
+	countIncidentsOutsideUS
+} from '../lib/utilities'
+import { ParentSize } from '@visx/responsive'
+import ChartDownloader from './ChartDownloader'
+import USMap from './USMap'
+
+export default ({
+	dataset,
+	title,
+	description,
```

Comment:
Speaking of accessibility: is this `description` field being used anywhere? Is this the kind of thing that we should be displaying as alt text or an ARIA attribute somewhere? Like I said, this is kind of out of my expertise.

Reply:
also my bad, thanks for catching!